docs/tutorial/performance.md

```diff
@@ -399,7 +399,7 @@ control over how resources are being loaded, consider investing in
 ### 7. Bundle your code
 
 As already pointed out in
-"[Loading and running code too soon](#2-loading-and-running-code-too-soon)",
+"[Loading and running code too soon](#2-avoid-loading-and-running-code-too-soon)",
```

Comment:
```suggestion
"[Avoid loading and running code too soon](#2-avoid-loading-and-running-code-too-soon)",
```

Needs to be updated to match the new wording.

Reply:
100% right. Thanks

Reply:
To be honest, I think we should be careful changing links. See also [here](https://github.com/electron/electron/pull/45241#discussion_r1920725711).